### ISSUE TITLE
Syndicate WikiApiary data to WikiIndex

### ISSUE THREAD

**Issue by jthingelstad** [WikiApiary/WikiApiary#75]
Rather than pulling data in this would update a template block on WikiIndex for wikis that WikiApiary is aware of.

The template can be managed using [mwparserfromhell](https://github.com/earwig/mwparserfromhell) to open page, add template specifically for WikiApiary and populate with key statistics.

Would require a bot account for WikiApiary on WikiIndex. See reference page for WikiApiary:

http://wikiindex.org/index.php?title=WikiApiary&action=edit

/cc @kghbln @MarkDilley

**Comment by kghbln** [comment]
Probably it will be best to put the statistics into a subpage and transclude the data into the basepage. Basically this is the system WikiAPIary uses, too. The "size" template could somehow be merged into the wiki template to make things easier to maintain. WikiFactor is a different thing, though. Then the frequency of updates should also be determinded. Probably a week or a fortnight is enough.

**Comment by jthingelstad** [comment]
Yes, for scheduling I would use the Day of week and Hour of day properties so each site would get updated no more than once a week, and all the edits would be smoothed out over the week so it wouldn't beat up WikiIndex with a burst of load.

Interesting idea to use a subpage, probably better and easier. I keep looking for an excuse to use [mwparserfromhell](https://github.com/earwig/mwparserfromhell) though. :-)
